From 9ae9302c8046c3dfee53b53abb3874a955e0606a Mon Sep 17 00:00:00 2001
From: Jakub Filak <jfilak@redhat.com>
Date: Mon, 3 Nov 2014 16:15:45 +0100
Subject: [LIBREPORT PATCH] rhtsupport: move RH Portal URL c. o. to Advanced
 section

It is unlikely that a regular user will ever want to edit that
configuration option.

Related to rhbz#1139987

Signed-off-by: Jakub Filak <jfilak@redhat.com>
---
 src/plugins/report_RHTSupport.xml.in | 14 ++++++++------
 1 file changed, 8 insertions(+), 6 deletions(-)

diff --git a/src/plugins/report_RHTSupport.xml.in b/src/plugins/report_RHTSupport.xml.in
index 9c0755a..aa3e886 100644
--- a/src/plugins/report_RHTSupport.xml.in
+++ b/src/plugins/report_RHTSupport.xml.in
@@ -11,12 +11,8 @@
     <minimal-rating>0</minimal-rating>
 
     <options>
-        <option type="text" name="RHTSupport_URL">
-            <_label>RH Portal URL</_label>
-            <allow-empty>no</allow-empty>
-            <_description>Address of the Red Hat support portal</_description>
-            <default-value>https://api.access.redhat.com/rs</default-value>
-        </option>
+        <import-event-options event="report_uReport"/>
+
         <option type="text" name="RHTSupport_Login">
             <_label>Username</_label>
             <_description>Red Hat customer user name</_description>
@@ -38,6 +34,12 @@
                 <_note-html>Submit &lt;a href="https://access.redhat.com/articles/642323"&gt;micro-report&lt;/a&gt; when creating a new case. </_note-html>
                 <default-value>yes</default-value>
             </option>
+            <option type="text" name="RHTSupport_URL">
+                <_label>RH Portal URL</_label>
+                <allow-empty>no</allow-empty>
+                <_description>Address of the Red Hat support portal</_description>
+                <default-value>https://api.access.redhat.com/rs</default-value>
+            </option>
             <option type="text" name="http_proxy">
                 <_label>HTTP Proxy</_label>
                 <allow-empty>yes</allow-empty>
-- 
1.8.3.1

